projects/plugins/jetpack/modules/custom-post-types/testimonial.php

```diff
@@ -416,6 +420,7 @@ public function updated_messages( $messages ) {
 	 * Change ‘Enter Title Here’ text for the Testimonial.
 	 *
 	 * @param string $title Placeholder text. Default 'Add title'.
+	 * @param string Replacement title.
```

Comment:
Oops, I made an error in my suggestion
```suggestion
	 * @return string Replacement title.
```